### PR TITLE
[11.0] migrate website_odoo_debranding

### PR DESCRIPTION
**Pull request by ddufresne**

### CODE REVIEW

File: website_odoo_debranding/__init__.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# -*- coding: utf-8 -*-
```
Comment (simahawk) [inline website_odoo_debranding/__init__.py]: remove encoding from *.py

File: website_odoo_debranding/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Tecnativa, S.L. - Vicent Cubells
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Remove odoo.com bindings on website",
+    "version": "11.0.1.0.0",
+    "author": "Tecnativa, "
+              "Odoo Community Association (OCA)",
+    "website": "https://www.tecnativa.com",
```
Comment (simahawk) [inline website_odoo_debranding/__manifest__.py]: https://github.com/OCA/website

File: website_odoo_debranding/README.rst [hunk 3]
```diff
@@ -0,0 +1,62 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
```
Comment (simahawk) [inline website_odoo_debranding/README.rst]: svg -> png (as svg is not supported in apps and images are not rendered)

**Review by simahawk — CHANGES_REQUESTED** [review]
small glitches

**Comment by pedrobaeza** [comment]
Check also existing warning:
```
2018-02-27 03:57:30,971 166 WARNING openerp_test odoo.addons.base.ir.ir_ui_view: Error-prone use of @class in view Remove Odoo Promotional Link (): use the hasclass(*classes) function to filter elements by their classes
2018-02-27 03:57:31,054 166 WARNING openerp_test odoo.addons.base.ir.ir_ui_view: Error-prone use of @class in view Remove Odoo Promotional Link (website_odoo_debranding.layout_footer_copyright): use the hasclass(*classes) function to filter elements by their classes
```

**Comment by ddufresne** [comment]
@simahawk fixed

**Review by simahawk — APPROVED** [review]

**Comment by simahawk** [comment]
@ddufresne sounds good. One last thing: squash your commits and reword to `[MIG][11.0] website_odoo_debranding` or similar. Tnx.

**Review by rgarnau — APPROVED** [review]
LGTM :+1:

**Comment by ddufresne** [comment]
@simahawk @rgarnau @pedrobaeza thanks for the review. sqash done

**Pull request merged**